### PR TITLE
Newsummary wip

### PR DESCRIPTION
**Pull request by OSPFNeighbour**

### CODE REVIEW

File: pages/scripts/summary.js [hunk 1]
```diff
@@ -4,10 +4,15 @@ var LighthouseJson = require('../lib/shared_json_code.js');
 var LighthouseChrome = require('../lib/shared_chrome_code.js');
 
 var $ = require('jquery');
+var _ = require('underscore');
+
 global.jQuery = $;
 var ElasticProgress = require('elastic-progress');
 var crossfilter = require('crossfilter');
 
+require('bootstrap');
```
Comment (sdunster) [inline pages/scripts/summary.js]: why require if you don't assign it?

File: pages/scripts/summary.js [hunk 2]
```diff
@@ -250,30 +333,35 @@ function HackTheMatrix(unit, host, token, progressBar) {
...
-      document.getElementById("flood").innerHTML = (jobs.Results.length && flood) ? (flood + "<h6>" + Math.round(flood / jobs.Results.length * 100) + "%</h6>"):(flood+"<h6>&ndash; %</h6>");
-      document.getElementById("rescue").innerHTML = (jobs.Results.length && rescue) ? (rescue + "<h6>" + Math.round(rescue / jobs.Results.length * 100) + "%</h6>"):(rescue+"<h6>&ndash; %</h6>");
-      document.getElementById("storm").innerHTML = (jobs.Results.length && storm) ? (storm + "<h6>" + Math.round(storm / jobs.Results.length * 100) + "%</h6>"):(storm+"<h6>&ndash; %</h6>");
-
-
-
+      var outstanding = newJob + activeJob + tskJob + refJob;
+      var completed = canJob + completeJob + finJob + rejJob;
+
+      _.each([
```
Comment (sdunster) [inline pages/scripts/summary.js]: this is fucking awesome :D

File: pages/scripts/summary.js (hunk 1 above)
Comment (OSPFNeighbour) [inline pages/scripts/summary.js]: modals need it, and it works without a assigment so I didnt bother.

File: pages/scripts/summary.js (hunk 2 above)
Comment (OSPFNeighbour) [inline pages/scripts/summary.js]: I cant tell if you are joking or not... given you wrote the each structure

File: pages/scripts/summary.js (hunk 1 above)
Comment (sdunster) [inline pages/scripts/summary.js]: add comment explaining this?

File: pages/scripts/summary.js (hunk 2 above)
Comment (sdunster) [inline pages/scripts/summary.js]: oh dammit I thought you wrote it. carry on then.

**Pull request merged**